api/design/type.go

```diff
@@ -282,3 +282,31 @@ var JWTAuth = JWTSecurity("jwt", func() {
 	Scope("rating:read", "Read-only access to rating")
 	Scope("rating:write", "Read and write access to rating")
 })
+
+var services = Type("services", func() {
```

Comment:
service?

Reply:
If we use `service` it gives an error because it goa already has an interface with `Service` and the error is `Service is redeclared

Reply:
Name it something else :smile:

Reply:
`services` -> `server` ?